docs/MapColonies/Raster/services/PolygonParts/profile_v1_0.md

```diff
@@ -16,13 +16,14 @@ tags:
 | **Polygon Part Queryable <br/> Attribute Name** | **Type** | **Description** | 
 | ----------- | ----------- | ----------- | 
 | geometry | GeoJson (geometry (Polygon,4326)) | The feature's geometry |
-| partId | integer | Internal unique part identifier (1-2147483647) |
-| recordId | UUID | Mapcolonies reaster catalog identifier for the layer |
+| id | UUID | Internal unique identifier |
+| partId | UUID | Part identifier |
```

Comment:
Please remove this. Remove from response too.
This isn't relevant for the end users.